src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/components/ResultItem.tsx

```diff
@@ -586,7 +586,7 @@ const TransportationLeg = ({
   return (
     <TransportationIconBox
       style={style}
-      mode={isLegFlexibleTransport(leg) ? 'flex' : leg.mode}
```

Comment:
I think it might be something else, this could break our flex. What it does that internally is to check if flexibleLineType is true or present.